.github/dependabot.yml

```diff
@@ -11,6 +16,18 @@ updates:
     directory: "/"
     schedule:
       interval: "weekly"
+    ignore:
+      - dependency-name: actions/cache
+      - dependency-name: actions/checkout
+      - dependency-name: actions/setup-python
+      # Managed by cisagov/skeleton-docker
+      # - dependency-name: actions/download-artifact
+      # - dependency-name: actions/github-script
+      # - dependency-name: actions/upload-artifact
+      # - dependency-name: docker/build-push-action
+      # - dependency-name: docker/login-action
+      # - dependency-name: docker/setup-buildx-action
+      # - dependency-name: docker/setup-qemu-action
```

Comment:
These lines should be uncommented.

Reply:
ah, i forgot this again. resolved with: [8a8482e](https://github.com/cisagov/con-pca-api/pull/637/commits/8a8482e07d79753227646172c44ce856ca7814f2)